### PR TITLE
New site design

### PR DESCRIPTION
**Pull request by ricklarsen**
Updated site design

@andygoodson @michaelmenousek @shaundmorris @codice/ddf-committers

### CODE REVIEW

**Comment by pklinef** [comment]
@ricklarsen Can you include some screenshots on this PR?

**Comment by ricklarsen** [comment]
screen shot 2017-01-31 at 8 09 27 am
screen shot 2017-01-31 at 8 09 49 am
screen shot 2017-01-31 at 8 10 08 am
screen shot 2017-01-31 at 8 10 31 am
screen shot 2017-01-31 at 8 10 47 am
screen shot 2017-01-31 at 8 11 09 am
screen shot 2017-01-31 at 8 11 40 am

File: about.html [hunk 1]
```diff
@@ -1,130 +1,96 @@
...
-<li>
-<a href="tools.html">Tools</a></li>
-<li class='last'>
-<a href="projects.html">Projects</a></li>
-  </ul>
-</nav>
+	<div class="container">
+		<div class="main-content-wrap">
+				<h2>About the Codice Foundation</h2>
+				<p>The origin of the Codice Foundation idea is a bit difficult to trace.  However, it's been in the back of many individuals minds and hidden in the activities of many programs and projects for a long time (e.g. ARPA).  In reality, the Codice Foundation is an sister to the active Mil-OSS Working Group that was formed in 2008.  As a result of many events and conversations the need for an Apache-like foundation that could serve the needs of federally-focused projects (and in particular in the Defense industry) was identified.  At the Mil-OSS WG4 in October of 2012, the plans for initiating Codice began to unfold.  Also, with the transitioning of the Ozone Widget Framework from closed GOTS to OSS the time was right to provide a broader scope to the Mil-OSS working group - and to provide tools and support to other projects who could benefit in the same ways.</p>
```
Comment (lessarderic) [inline about.html]: Typo: is ***a*** sister? Also, we use _need_ twice in the same sentence.

File: index.html [hunk 2]
```diff
@@ -1,147 +1,178 @@
...
+					<div class="project">
+						<img class="project-logo" src="images/logo-codice-imaging.png" title="Codice Imaging" alt="Codice Imaging">
+						<p>This provides Pure Java National Imagery Transmission Format (NITF) file support.</p>
+						<a class="btn-secondary" href="https://github.com/codice/imaging-nitf" target="_blank">View Project &raquo;</a>
+					</div>
+				</div>
+				<div class="project-wrap project-3">
+					<div class="project">
+						<img class="project-logo" src="images/logo-alliance.png" title="Alliance" alt="Alliance">
+						<p>An Intelligence, surveillance and reconnaissance (ISR) variant to DDF with geospatial capabilities.</p>
```
Comment (lessarderic) [inline index.html]: Should we capitalize _surveillance_ and _reconnaissance_ since it's an acronym?

File: template.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+					</div>
+					<div class="nav-item"><a href="tools.html">Tools</a></div>
+				</nav>
+			</div>
+		</div>
+	</div>
+
+	<div class="container">
+		<div class="main-content-wrap">
+				This is where content goes.
```
Comment (lessarderic) [inline template.html]: Should this be a `<!-- comment -->`?

**Review by lessarderic — APPROVED** [review]

File: template.html (hunk 3 above)
Comment (TeresaHudson) [reply template.html]: It's not used in the actual website, so it doesn't matter. It's just there as a bare bones template. Whatever is in there will be deleted anyway.

File: index.html (hunk 2 above)
Comment (TeresaHudson) [reply index.html]: Absolutely.

File: about.html (hunk 1 above)
Comment (TeresaHudson) [reply about.html]: Yes, and we can change the sentence to "As a result of many events and conversations the necessity for an Apache-like foundation..."

File: agreements.html [hunk 4]
```diff
@@ -1,164 +1,49 @@
...
-<p>The Individual Contributor License Agreement (ICLA) can be <a href="https://docs.google.com/document/d/1GFO28twAu1uDh-AY8K9_mOcMH67XqHN6s2RDu11sNpQ/pub" title="https://docs.google.com/document/d/1GFO28twAu1uDh-AY8K9_mOcMH67XqHN6s2RDu11sNpQ/pub" target="_blank">downloaded here</a>.&#160; PDF version available <a title="Icla.Txt" href="pdfs/icla.txt.pdf">here</a>.</p>
-<iframe src="https://docs.google.com/document/d/1GFO28twAu1uDh-AY8K9_mOcMH67XqHN6s2RDu11sNpQ/pub?embedded=true" height="400px" width="800px"></iframe>
--->
-<p>The Individual Contributor License Agreement (ICLA) can be <a href="pdfs/icla.pdf" title="pdfs/icla.pdf" target="_blank">downloaded here</a>.</p>
-<iframe src="pdfs/icla.txt?embedded=true" height="400px" width="800px"></iframe>
+	<div class="container">
+		<div class="main-content-wrap">
+			<h2>License and Contributor Agreements</h2>
+			<h3>License(s)</h3>
+			<p>The Codice Foundation itself doesn't not maintain its own license.  However, in order for projects to be entered into incubation they must already be licensed with an <a href="http://opensource.org/licenses" target="_blank">approved OSI license.</a></p>
```
Comment (shaundmorris) [inline agreements.html]: `doesn't not` sounds a bit awkward can we clarify this?

**Review by shaundmorris — APPROVED** [review]

**Pull request merged**